components/Snackbar/src/MDCSnackbarMessageView.m

```diff
@@ -214,35 +214,34 @@ @implementation MDCSnackbarMessageView {
 }
 
 - (instancetype)initWithFrame:(CGRect)frame {
-  self = [super initWithFrame:frame];
+  return [self initWithMessage:nil
+                dismissHandler:nil
+               snackbarManager:MDCSnackbarManager.defaultManager];
+}
+
+- (instancetype)initWithMessage:(MDCSnackbarMessage *)message
+                 dismissHandler:(MDCSnackbarMessageDismissHandler)handler
+                snackbarManager:(MDCSnackbarManager *__weak)manager {
+  self = [super initWithFrame:CGRectZero];
 
   if (self) {
     _snackbarMessageViewShadowColor =
-        MDCSnackbarManager.snackbarMessageViewShadowColor ?: UIColor.blackColor;
+        manager.snackbarMessageViewShadowColor ?: UIColor.blackColor;
     _snackbarMessageViewBackgroundColor =
-        MDCSnackbarManager.snackbarMessageViewBackgroundColor ?: MDCRGBAColor(0x32, 0x32, 0x32, 1);
+        manager.snackbarMessageViewBackgroundColor ?: MDCRGBAColor(0x32, 0x32, 0x32, 1);
     _messageTextColor =
-        MDCSnackbarManager.messageTextColor ?: UIColor.whiteColor;
+        manager.messageTextColor ?: UIColor.whiteColor;
     _buttonTitleColors = [NSMutableDictionary dictionary];
     _buttonTitleColors[@(UIControlStateNormal)] =
-        [MDCSnackbarManager buttonTitleColorForState:UIControlStateNormal] ?:
-            MDCRGBAColor(0xFF, 0xFF, 0xFF, 0.6f);
+        [manager buttonTitleColorForState:UIControlStateNormal] ?:
+        MDCRGBAColor(0xFF, 0xFF, 0xFF, 0.6f);
     _buttonTitleColors[@(UIControlStateHighlighted)] =
-        [MDCSnackbarManager buttonTitleColorForState:UIControlStateHighlighted] ?:
-            UIColor.whiteColor;
+        [manager buttonTitleColorForState:UIControlStateHighlighted] ?:
+        UIColor.whiteColor;
     _mdc_adjustsFontForContentSizeCategory =
-        MDCSnackbarManager.mdc_adjustsFontForContentSizeCategory;
-    _messageFont = MDCSnackbarManager.messageFont;
-    _buttonFont = MDCSnackbarManager.buttonFont;
-  }
-
-  return self;
-}
-
-- (instancetype)initWithMessage:(MDCSnackbarMessage *)message
-                 dismissHandler:(MDCSnackbarMessageDismissHandler)handler {
-  self = [super init];
-  if (self) {
+        manager.mdc_adjustsFontForContentSizeCategory;
```

Comment:
indentation

Reply:
Could you please clarify? The indentation (4 spaces) looks correct for a continued statement.

Reply:
it's a continued statement from `if (self) {` ?
meaning you wanted it to be `if (self) { manager.mdc_adjustsFontForContentSizeCategory;` but it was too long to fit in one line?

Reply:
oh shoot, i didn't see that line back there, my bad. ignore